tools/accuracy_checker/accuracy_checker/adapters/retinanet.py

```diff
@@ -333,3 +335,255 @@ def regress_boxes(self, boxes, deltas):
+            'anchor_size': NumberField(optional=True, default=4, description='anchor box size'),
+            'total_size': NumberField(
+                optional=True, default=100, value_type=int, min_value=1, description='final number of boxes'
+            ),
+            'pre_nms_top_k': NumberField(
+                optional=True, value_type=int, min_value=1, default=5000,
+                description='number of keep top by score boxes before nms'),
+            'score_threshold': NumberField(
+                value_type=float, min_value=0, max_value=1, default=0.05, description='scores threshold'
+            ),
+            'nms_threshold': NumberField(
+                value_type=float, min_value=0, max_value=1, default=0.5, description='nms threshold'
+            )
+
+        })
+        return params
+
+    def configure(self):
+        self.loc_out = self.get_value_from_config('boxes_outputs')
+        self.cls_out = self.get_value_from_config('class_outputs')
+        self.min_level = self.get_value_from_config('min_level')
+        self.max_level = self.get_value_from_config('max_level')
+        self.aspect_ratios = self.get_value_from_config('aspect_ratios')
+        self.anchor_size = self.get_value_from_config('anchor_size')
+        self.num_scales = self.get_value_from_config('num_scales')
+        self.max_total_size = self.get_value_from_config('total_size')
+        self.nms_iou_threshold = self.get_value_from_config('nms_threshold')
+        self.score_threshold = self.get_value_from_config('score_threshold')
+        self.pre_nms_num_boxes = self.get_value_from_config('pre_nms_top_k')
+
+    def _generate_anchor_boxes(self, image_size):
+        boxes_all = []
+        for level in range(self.min_level, self.max_level + 1):
+            boxes_l = []
+            for scale in range(self.num_scales):
+                for aspect_ratio in self.aspect_ratios:
+                    stride = 2 ** level
+                    intermediate_scale = 2 ** (scale / float(self.num_scales))
+                    base_anchor_size = self.anchor_size * stride * intermediate_scale
+                    aspect_x = aspect_ratio ** 0.5
+                    aspect_y = aspect_ratio ** -0.5
+                    half_anchor_size_x = base_anchor_size * aspect_x / 2.0
+                    half_anchor_size_y = base_anchor_size * aspect_y / 2.0
+                    x = np.arange(stride / 2, image_size[1], stride)
+                    y = np.arange(stride / 2, image_size[0], stride)
+                    xv, yv = np.meshgrid(x, y)
+                    xv = np.reshape(xv, -1)
+                    yv = np.reshape(yv, -1)
+                    boxes = np.stack([
+                        yv - half_anchor_size_y, xv - half_anchor_size_x,
+                        yv + half_anchor_size_y, xv + half_anchor_size_x
+                    ], axis=1
+                    )
```

Comment:
```suggestion
                    ], axis=1)
```
Maybe it'll solve pylint error

Reply:
ok, updated, thanks. hope, it makes linter happy:)